package.json

```diff
@@ -23,7 +23,7 @@
     "start:dev:renderer-reader": "concurrently --kill-others \"npm run start:dev:renderer\" \"npm run start:dev:reader\"",
     "start:dev:renderer": "webpack-dev-server --config webpack.config.renderer.js",
     "start:dev:reader": "webpack-dev-server --config webpack.config.reader.js",
-    "start:dev:main:electron": "cross-env DEBUG=r2:*,readium-desktop:* NODE_ENV=development electron .",
+    "start:dev:main:electron": "cross-env DEBUG=readium-desktop:main:redux:sagas:reader,readium-desktop:common:rectangle NODE_ENV=development electron .",
```

Comment:
Is that change intentional? This looks very specific to this PR.

General note: I imagine the use-case for this (local) change was to decrease the default verbosity of `debug()` console output, and narrow it down to PR-specific logging. That's a pretty common use-case for any developer. I wonder how we could set the `DEBUG` environment variable **outside** of `npm run start:dev` (i.e. on the calling side) so that it takes precedence over the `DEBUG` environment variable set by `cross-env` (for example, to remove all `r2:*` console output by using the negative / minus sign `-`). Unfortunately I think this is not possible, because `cross-env` has a higher priority. https://github.com/kentcdodds/cross-env

Reply:
done